serverless/aws/package.json

```diff
@@ -7,12 +7,13 @@
     "@types/node": "^18.11.2",
     "esbuild": "^0.20.0",
     "tsx": "^4.7.0",
-    "typescript": "^4.8.4"
+    "typescript": "^4.8.4",
+    "yaml-cfn": "^0.3.2"
   },
   "private": true,
   "scripts": {
     "tsc": "tsc --noEmit --watch",
-    "build": "esbuild src/index.ts --target=es2020 --outfile=dist/index.mjs --format=esm --bundle --platform=node --target=node18 --external:@aws-sdk/client-s3 --external:@aws-sdk/node-http-handler --banner:js=//$(git describe --always) && cd dist && zip lambda_function.zip index.mjs",
+    "build": "esbuild src/index.ts --target=es2020 --minify --outfile=dist/index.js --format=cjs --bundle --platform=node --target=node20 --external:@aws-sdk/client-s3 --external:@aws-sdk/node-http-handler --banner:js=//sha:$(git describe --always)",
```

Comment:
Any thoughts on including the template as part of the distribution for PMTiles?

It would be great to be able load the compiled yaml directly in JS ('import { pmtilesCFN } from "pmtiles"'). This could be used alongside CfnInclude to embed this infra in existing CDK stacks. Otherwise, clients will need to manually copy the template.

Reply:
do you mean as part of the `pmtiles` NPM package, that can be included as a AWS CDK dependency?

Originally I kept the Cloudflare and AWS specific code out of the main JS package to reduce versioning churn and dependencies, but since those are stabilizing we can consider this in a next major version.

Reply:
Yup, any NPM package can be used in typescript based CDK projects.